wagtail/bin/wagtail.py

```diff
@@ -30,14 +36,17 @@ def create_project(parser, options, args):
                      "Python module and cannot be used as a project "
                      "name. Please try another name." % project_name)
 
-    print("Creating a Wagtail project called %(project_name)s" % {'project_name': project_name})  # noqa
+    if project_template:
+        print("Creating a Wagtail project called %(project_name)s using template from %(project_template)s" % {'project_name': project_name, 'project_template': project_template})  # noqa
+    else:
+        print("Creating a Wagtail project called %(project_name)s" % {'project_name': project_name})  # noqa
 
     # Create the project from the Wagtail template using startapp
 
     # First find the path to Wagtail
     import wagtail
     wagtail_path = os.path.dirname(wagtail.__file__)
-    template_path = os.path.join(wagtail_path, 'project_template')
+    template_path = os.path.join(wagtail_path, 'project_template' if project_template is None else project_template)
```

Comment:
Is this addition really meant to be inside the `os.path.join(...)`? That means that your template path has to be specified relative to the Wagtail library root, which surely isn't what you want...